src/main/resources/templates/search.html

```diff
@@ -18,9 +18,8 @@ <h4>Search for your car</h4>
 	    <div class="form-group" id="carMake">
 	        <label for="carMake">Car Make & Model</label>
 	        <br/>
-<!-- 	Commented Out - Creating drop down list        <input class="form-control" type="text" th:field="*{carMake}" id="carMake" placeholder="Make">  -->
 		<select name="carModel" id="carModel">
-			 <option disabled selected value> -- select an option -- </option>
+			 <option disabled selected value=""> -- select an option -- </option>

```

Comment:
Although this is technically fine, having the value attribute with no declaration will throw warnings in both eclipse and browsers. I added a blank value to satisfy the html.